cmd/gazelle/update-repos.go

```diff
@@ -108,20 +131,34 @@ func updateRepos(args []string) error {
 	}
 	uc := getUpdateReposConfig(c)
 
-	workspacePath := filepath.Join(c.RepoRoot, "WORKSPACE")
-	f, err := rule.LoadWorkspaceFile(workspacePath, "")
+	var f *rule.File
+	var path string
+	if uc.macroFileName == "" {
+		path = filepath.Join(c.RepoRoot, "WORKSPACE")
+		f, err = rule.LoadWorkspaceFile(path, "")
+	} else {
+		path = uc.macroFileName
+		f, err = rule.LoadMacroFile(path, "", uc.macroDefName)
```

Comment:
If the file doesn't exist, we should create a new one.

Reply:
Fixed and tested with TestImportReposFromDepToEmptyMacro